allocations/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+import json
+import logging
+from django.conf import settings
+
+logger = logging.getLogger('allocations')
+
+class Allocation(models.Model):
+    STATUS = (
+        ('Inactive', 'Expired or overused'),
+        ('Active', 'Active'),
+        ('Pending', 'Waiting for review decision'),
+        ('Rejected', 'Rejected'),
+        ('Approved', 'Approved but not active')
+    )
+    project_charge_code = models.CharField(max_length=50,blank=False)
+    status = models.CharField(max_length=50, blank=False,choices=STATUS)
+    justification = models.CharField(max_length=500,null=True)
+    requestor = models.ForeignKey(settings.AUTH_USER_MODEL,related_name='allocation_requestor', null=True)
```

Comment:
It's really nice to be able to do this. I suppose it means all reviewers have to have Chameleon accounts, but that is a very reasonable requirement.